client/src/Widgets/ContentCard.tsx

```diff
@@ -158,7 +158,18 @@ export default forwardRef(function ContentCard(
                   autoFocus={autoFocusTitle}
                   onFocus={(e) => e.target.select()}
                   onChange={(e) => setCardTitle(e.target.value)}
-                  onBlur={saveUpdatedTitle}
+                  onBlur={(e) => {
+                    saveUpdatedTitle();
+                    // prevent click default/propagation behavior one time (aka right now as user is clicking to blur input)
+                    document.addEventListener(
```

Comment:
Given the solution of line 175, below, to trigger `blur` when pressing `Enter`, adding this event listener now gives undesired behavior after pressing `Enter` on a title: now after pressing enter, the next click is ignored! You'll need to separate out the logic to make sure this doesn't happen!

Do we want an `<EditableTitle>` here, too, like in `<ActivityTable>`? Anything you can do to consolidate code between the list view and the card view will make it easier to maintain and make sure we get consistent behavior between the two views.